src/osd/OSD.cc

```diff
@@ -1882,14 +1882,21 @@ void OSDService::queue_scrub_next_chunk(PG *pg, Scrub::scrub_prio_t with_priorit
   queue_scrub_event_msg<PGScrubGetNextChunk>(pg, with_priority);
 }
 
-void OSDService::queue_for_pg_delete(spg_t pgid, epoch_t e)
+void OSDService::queue_for_pg_delete(spg_t pgid, epoch_t e, int64_t num_objects)
 {
   dout(10) << __func__ << " on " << pgid << " e " << e  << dendl;
+  uint64_t cost_for_queue = [this, num_objects] {
+    if (cct->_conf->osd_op_queue == "mclock_scheduler") {
```

Comment:
Please rebase and modify this line since the check for scheduler type is changed according to the recently merged PR: https://github.com/ceph/ceph/pull/53524.

Reply:
@amathuria LGTM.